src/main/java/vic/cinema/model/Ticket.java

```diff
@@ -0,0 +1,50 @@
+package vic.cinema.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "tickets")
+public class Ticket {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToOne
+    private MovieSession movieSession;
+    @OneToOne
```

Comment:
якщо я правильно зрозумів Софію на Q&A багато тікетів відносяться до однієї мувісесії, і багато тікетів до одного юзера
```suggestion
    @ManyToOne
```

Reply:
Done